FirebaseAuthUI/FUIAuth.m

```diff
@@ -186,19 +187,19 @@ - (void)signInWithProviderUI:(id<FUIAuthProvider>)providerUI
       }
 
       if (result) {
-        result(user, error);
+        result(authResult.user, error);
       }
 
       if (error) {
-        [self invokeResultCallbackWithUser:user error:error];
+        [self invokeResultCallbackWithAuthDataResult:authResult error:error];
```

Comment:
If an error exist we shouldn't pass authResult here which should ideally be nil, but at this level we do not know.

Reply:
Done